candi/bashible/common-steps/all/039_add_kubectl_completion.sh.tpl

```diff
@@ -12,14 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-sed -i 's/\# \"\\e\[5~\": history-search-backward/\"\\e\[5~\": history-search-backward/' /etc/inputrc
-sed -i 's/^\# \"\\e\[6~\": history-search-forward/\"\\e\[6~\": history-search-forward/' /etc/inputrc
+if grep -qF '# "\e[5~": history-search-backward' /etc/inputrc; then
+  sed -i 's/\# \"\\e\[5~\": history-search-backward/\"\\e\[5~\": history-search-backward/' /etc/inputrc
+fi
+if grep -qF '# "\e[6~": history-search-forward' /etc/inputrc; then
+  sed -i 's/^\# \"\\e\[6~\": history-search-forward/\"\\e\[6~\": history-search-forward/' /etc/inputrc
+fi
 
-sed -i 's/\#force_color_prompt=yes/force_color_prompt=yes/' /root/.bashrc
-sed -i 's/01;32m/01;31m/' /root/.bashrc
+if grep -qF '#force_color_prompt=yes' /root/.bashrc; then
+  sed -i 's/\#force_color_prompt=yes/force_color_prompt=yes/' /root/.bashrc
+fi
+if grep -qF '01;32m' /root/.bashrc; then
+  sed -i 's/01;32m/01;31m/' /root/.bashrc
+fi
 
 mkdir -p /etc/bash_completion.d
 kubectl completion bash >/etc/bash_completion.d/kubectl
```

Comment:
we need to check if /etc/bash_completion.d/kubectl already exists

Reply:
Fixed